### PR TITLE
Adding missing curly braces to sample webpack config

### PR DESCRIPTION
**Pull request by CarlosTorrecillas**
To fix ERROR "unexpected token )"

### CODE REVIEW

**Comment by filipesilva** [comment]
Hi, we can't merge this without a signed CLA. Can you have a look at it please?

**Comment by CarlosTorrecillas** [comment]
I did register I while ago...

**Comment by filipesilva** [comment]
@CarlosTorrecillas thanks for checking the CLA. With that out of the way, I had a look at the CI failure. It is due to your commit message actually. We check commit messages to make changelogs from them so they need to be a specific format.

Could you change it to `docs: Adding missing curly braces to sample webpack config` please?

File: docs/documentation/stories/universal-rendering.md [hunk 1]
```diff
@@ -336,6 +336,7 @@ module.exports = {
     new webpack.ContextReplacementPlugin(
       /(.+)?express(\\|\/)(.+)?/,
       path.join(__dirname, 'src'),
+      {}
```
Comment (filipesilva) [inline docs/documentation/stories/universal-rendering.md]: Would it not be better to remove the command and third argument altogether, or would this be incorrect?

**Comment by clydin** [comment]
Closing due to inactivity.  If the change is still of interesting, please open a new PR.
Thank you for the contribution.

**Pull request closed without merging**